tests/conftest.py

```diff
@@ -224,19 +235,16 @@ def structurally_valid_vote():
 
 
 @pytest.fixture
-def inputs(user_pk):
-    from bigchaindb import Bigchain
+def genesis_block(b):
+    return b.create_genesis_block()
```

Comment:
Wondering if it might be worth keeping the try/except. Leaning on not, since you probably screwed up if you ended up trying to create two genesis blocks in a test.